code/modules/mob/living/simple_animal/abnormality/_abnormality.dm

```diff
@@ -91,6 +91,31 @@
 	//Abnormaltiy portrait, updated on spawn if they have one.
 	var/portrait = "UNKNOWN"
 
+	// secret skin variables ahead
+
+	/// Toggles if the abnormality has a secret form, if not FALSE this variable will control the chance for it to appear
+	var/secret_chance = FALSE
```

Comment:
Don't use FALSE/TRUE defines unless you are intending to use the variable as boolean.

Reply:
This is actually used later for the 1% chance. It's a Yes or no.